recipe/meta.yaml

```diff
@@ -147,8 +162,12 @@ outputs:
     test:
       commands: []
     about:
-      summary: >-
-        The NCCL communications layer module for NVIDIA cuDSS
+      summary: Install this package to enable NCCL for cuDSS.
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_file: LICENSE
+      description: >-
+        This is a runtime package only. Developers should install libcudss-dev to build with
```

Comment:
If my check was correct and that package is empty, we should maybe point to the non-empty one (which is libcudss-commlayer-nccl{{ somajor }})?
Or are these two somehow aliased?

(same question for nccl)

Reply:
I mean, my only concern here is that if anyone sets this package as a dependency, will it break things (if it is empty)? If so, saying that this is "a runtime package" is a bit misleading.